Tests/FluentAssertions.Equivalency.Specs/DataColumnSpecs.cs

```diff
@@ -77,6 +78,43 @@ public void When_DataColumn_has_changes_but_is_excluded_it_should_succeed()
             .ExcludingColumn(dataColumn2));
     }
 
+    [Fact]
+    public void When_DataColumn_has_changes_but_is_excluded_as_params_it_should_succeed()
```

Comment:
🌱 We really need to start improving those test names, but that's besides the point of this PR.